airflow/dag_processing/processor.py

```diff
@@ -473,13 +473,21 @@ def manage_slas(self, dag: DAG, session: Session = None) -> None:
             notification_sent = False
             if dag.sla_miss_callback:
                 # Execute the alert callback
-                self.log.info("Calling SLA miss callback")
-                try:
-                    dag.sla_miss_callback(dag, task_list, blocking_task_list, slas, blocking_tis)
-                    notification_sent = True
-                except Exception:
-                    Stats.incr("sla_callback_notification_failure")
-                    self.log.exception("Could not call sla_miss_callback for DAG %s", dag.dag_id)
+                callbacks = (
+                    dag.sla_miss_callback
+                    if isinstance(dag.sla_miss_callback, list)
+                    else [dag.sla_miss_callback]
+                )
+                for callback in callbacks:
+                    self.log.info("Calling SLA miss callback %s", callback)
+                    try:
+                        callback(dag, task_list, blocking_task_list, slas, blocking_tis)
+                        notification_sent = True
+                    except Exception:
+                        Stats.incr("sla_callback_notification_failure")
+                        self.log.exception(
+                            "Could not call sla_miss_callback(%s) for DAG %s", callback, dag.dag_id
```

Comment:
```suggestion
                            "Could not call sla_miss_callback %s for DAG %s",
                            qualname(callback),
                            dag.dag_id,
```

Probably more readable? `qualname` is in `airflow.utils.module_loading`.

Reply:
`qualname` looks like it's doing a lot, gives the importable name of the function

Reply:
I think I prefer the previous implementation

Reply:
WDYT @uranusjr , should I use `inspect` or leave it as it was before?

Reply:
It’s OK to leave it as is if you prefer, this is a log message and there’s no one right answerr

Reply:
Cool